main.cpp

```diff
@@ -49,87 +49,97 @@ arm.extend();                           // 3)
 arm.flex();                             // 3)
 //  action 3:   the arm rotates conter-clockwise
 arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
-
+
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       dog
+//  action 1:   the dog barks
+dog.bark();
+//  action 2:   the dog sleeps
+dog.sleep();
+//  action 3:   the dog eats
+dog.eat();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       computer
+//  action 1:   the computer starts Windows
+computer.startWindows();
+//  action 2:   the computer runs startup applications
+computer.runStartupApps();
+//  action 3:   the computer restarts
+computer.restart();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       musicPlayer
+//  action 1:   the music player plays a song
+musicPlayer.play();
+//  action 2:   the music player stops a song
+musicPlayer.stop();
+//  action 3:   the music player skips a song
+musicPlayer.skip();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       mobilePhone
+//  action 1:   the mobile phone turns on
+mobilePhone.on();
+//  action 2:   the mobile phone turns off
+mobilePhone.off();
+//  action 3:   the mobile phone opens messages app
+mobilePhone.openMessages();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       monster
+//  action 1:   monster attacks player
+monster.attack();
+//  action 2:   monster blocks attack
+monster.block();
+//  action 3:   monster runs away
+monster.flee();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.